packages/client/components/TeamPrompt/TeamPromptWorkDrawer.tsx

```diff
@@ -39,12 +43,22 @@ const TeamPromptWorkDrawer = (props: Props) => {
     `,
     meetingRef
   )
+  const atmosphere = useAtmosphere()
+
+  useEffect(() => {
+    SendClientSegmentEventMutation(atmosphere, 'Your Work Drawer Opened', {
+      teamId: meeting.teamId,
+      meetingId: meeting.id,
+      source: 'impression'
```

Comment:
-1 If I have the discussion panel opened, and then I clicked "Your work" from the top bar, I got two `Your Work Drawer Opened` events, one with the source `'top bar'`, the other with the source `'impression'`. I don't think the latter should be emitted?

Reply:
This is tricky, since we need the `'impression'` trigger when it's open by default, and we need the `'top bar'` trigger when the user clicks to open it.

What if we changed the `impression` version to a different event like `'Your Work Drawer Impression'`? That way we can still do analysis based on when a user sees the work drawer.

Reply:
Yeah that works!